assets/training/finetune_acft_multimodal/environments/acpt_multimodal/context/Dockerfile

```diff
@@ -11,6 +11,5 @@ RUN pip install -r requirements.txt --no-cache-dir
 # Vulnerability fix
 RUN pip install datasets==2.15.0
 RUN pip install pyarrow==14.0.1
-RUN pip install urllib3==2.2.0
 RUN pip install cryptography==42.0.5
```

Comment:
@shubhamiit - you can also remove the other ones, after checking in the base image.

Reply:
Datasets lib is pinned to older version in acpt-multimodal pkg, need to remove from there. Can't remove it here before removing it from there.